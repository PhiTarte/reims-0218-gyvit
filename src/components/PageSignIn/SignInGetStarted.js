//TODO CREATE SIGN IN TITLE
import React from "react"
import { NavLink } from "reactstrap"

const SignInGetStarted = () => (
  <div style={{ marginTop: "2em" }}>
    <p>
      Don't have an account ?
      <NavLink
        href="#"
        style={{
          display: "inline"
        }}
      >
        Get started
      </NavLink>
    </p>
  </div>
)

export default SignInGetStarted
