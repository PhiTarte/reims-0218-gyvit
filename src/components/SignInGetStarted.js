//TODO CREATE SIGN IN TITLE
import React from "react"
import { NavLink } from "reactstrap"

const SignInGetStarted = () => (
  <div style={{ "margin-top": "120px" }}>
    <p>
      Don't have an account ?
      <NavLink
        href="#"
        style={{
          display: "inline"
        }}
      >
        Get started
      </NavLink>
    </p>
  </div>
)

export default SignInGetStarted
