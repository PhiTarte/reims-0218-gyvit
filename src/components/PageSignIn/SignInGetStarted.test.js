//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import SignInGetStarted from "./SignInGetStarted"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<SignInGetStarted />, div)
  ReactDOM.unmountComponentAtNode(div)
})
