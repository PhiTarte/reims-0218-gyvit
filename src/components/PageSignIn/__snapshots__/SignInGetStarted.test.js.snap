// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div
  style={
    Object {
      "marginTop": "120px",
    }
  }
>
  <p>
    Don't have an account ?
    <a
      className="nav-link"
      href="#"
      onClick={[Function]}
      style={
        Object {
          "display": "inline",
        }
      }
    >
      Get started
    </a>
  </p>
</div>
`;
